packages/app/src/app/components/WorkspaceSetup/steps/Plans.tsx

```diff
@@ -386,61 +343,64 @@ const PlanAndPricing: React.FC<{ plan: PricingPlan }> = ({ plan }) => {
   );
 };
 
-const PlanCredits: React.FC<{ plan: PricingPlan }> = ({ plan }) => (
-  <Stack direction="vertical" align="center" css={{ height: '35px' }}>
-    {plan.credits > 0 &&
-      (plan.creditsNote ? (
-        <Tooltip label={plan.creditsNote}>
-          <Text
-            css={{
-              borderBottom: '1px dotted currentColor',
-              marginBottom: '5px !important',
-            }}
-          >
-            {plan.credits} credits included
-          </Text>
-        </Tooltip>
-      ) : (
-        <Text>{plan.credits} credits included</Text>
-      ))}
-    {plan.additionalCreditsCost ? (
-      <Text size={2} color="#4E3BB0" css={{ textAlign: 'center' }}>
-        Need more? ${plan.additionalCreditsCost} $/cr
+const PlanFeatures: React.FC<{
+  features: string[];
+  heading?: string;
+  secondaryColor?: string;
+  itemIcon?: IconNames;
+  includeTooltips?: boolean;
+}> = ({
+  features,
+  heading,
+  secondaryColor = '#5c5c5c',
+  itemIcon = 'simpleCheck',
+  includeTooltips = false,
+}) => (
+  <Stack
+    direction="vertical"
+    align="flex-start"
+    css={{ width: '100%' }}
+    gap={4}
+  >
+    {heading && (
+      <Text css={{ marginLeft: '28px' }} size={3} color={secondaryColor}>
+        {heading}
       </Text>
-    ) : null}
+    )}
+    <Stack as="ul" css={{ padding: 0, margin: 0 }} gap={4} direction="vertical">
+      {features.map(feature => (
+        <Stack as="li" key={feature.toString()} gap={2} align="flex-start">
+          <Icon color={secondaryColor} name={itemIcon} size={20} />
+          {includeTooltips ? (
```

Comment:
To simplify it, we could only use `TextWithTooltips` as it defaults to `Text` in case there's no tooltip, then you could drop `includeTooltips` prop too.